Example/SnapshotTests/BPKCalendarSnapshotTest.m

```diff
@@ -44,6 +45,7 @@ - (void)setUp {
     self.date1 = [NSDate dateWithTimeIntervalSince1970:2175785688];
     self.date2 = [NSDate dateWithTimeIntervalSince1970:2176044888];
     self.isColoringDates = NO;
+    self.isShowingPrices = NO;
```

Comment:
This will break if we ever run these tests concurrenctly(is that even possible?)

Reply:
It seemed to work before for `isColouringDates` so I assume it's fine.
I'm pretty sure the tests never run concurrently.

Reply:
It's probably the reverse i.e. that both would break but we don't run concurrently.